scripts/zkai_tablero_hybrid_zkml_boundary_gate.py

```
@@ -0,0 +1,799 @@
+    if len(examples) != 5:
+        raise TableroHybridBoundaryError("boundary example inventory drift")
+    seen = set()
+    for row in examples:
+        validate_boundary_statement(row)
+        statement_id = row["statement_id"]
+        if statement_id in seen:
+            raise TableroHybridBoundaryError("duplicate statement id")
+        seen.add(statement_id)
+    summary = require_dict(payload.get("summary"), "summary")
+    if summary.get("jstprove_statement_envelope_mutations_rejected") != 13:
+        raise TableroHybridBoundaryError("JSTprove statement-envelope mutation drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)